product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_10_2_Proxy_AlterPayloadWithInlineJavaScript.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shall we add the license header to all files that are missing it?

Reply:
Done